spec/controllers/early_allocations_controller_spec.rb

```diff
@@ -93,14 +93,17 @@
         s1_boolean_param_names.each do |field|
           s1_boolean_params[field] = 'true'
 
-          post :create, params: { prison_id: prison,
-                                  prisoner_id: nomis_offender_id,
-                                  early_allocation: date_params.merge(s1_boolean_params) }
+          expect {
+            post :create, params: { prison_id: prison,
+                                    prisoner_id: nomis_offender_id,
+                                    early_allocation: date_params.merge(s1_boolean_params) }
+          }.to change(EmailHistory, :count).by(1)
           assert_template('eligible')
 
           expect(early_allocation.prison).to eq(prison)
           expect(early_allocation.created_by_firstname).to eq(first_pom.firstName)
           expect(early_allocation.created_by_lastname).to eq(first_pom.lastName)
+          expect(early_allocation.created_within_referral_window).to eq(true)
```

Comment:
Is there any value in having a context to assert that `created_within_referral_window` is `false` and an email is not sent when `> 18 months`?

I know we have a split between `<= 18 months` and `> 18 months` in the feature spec – would the same thing would make sense here?

Just a question – not a request.